taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -0,0 +1,94 @@
+import { Typography, Paper, TextField, Stack, Box } from "@mui/material";
+import ISO6391 from 'iso-639-1';
+
+// Sub-component for rendering translation of an "entry"
+
+const ListTranslations = ({ nodeObject, setNodeObject }) => {
+    let RenderedTranslations = {}
+
+    Object.keys(nodeObject).forEach((key) => {
+        
+        // Get all tags and its corresponding language code
+        // Tagids need to be recomputed, so shouldn't be rendered
+        // Main language isn't considered, since it's rendered separately
+ 
+        if (key.startsWith('tags') && 
+            !key.endsWith(nodeObject.main_language) && 
+            !key.includes('ids')) {
+
+                // Slice the language code
+                let languageCode = key.slice(-2);
+                RenderedTranslations[languageCode] = nodeObject[key]
+            }
+    })
+    
+    // Helper function used for changing state
+    function changeData(key, index, value) {
+        key = 'tags_' + key;
+        const duplicateData = {...nodeObject};
+        duplicateData[key][index] = value;
+        setNodeObject(duplicateData);
+    }
+
+    return ( 
+        <Box className="translations">
+            {/* Title */}
+            <Typography sx={{mt: 4, mb: 1}} variant='h5' component={'div'}>Translations</Typography>
+            {/* Main Language */}
+            <Typography variant='h6'>
+                { ISO6391.getName(nodeObject.main_language) }
+            </Typography>
+            {/* Render main language tags */}
+            <Typography variant='h6'> 
+                {
+                    nodeObject["tags_"+nodeObject['main_language']].map((tag, index) => {
+                        return (
+                            <Paper key={index} component={Stack} direction="column" sx={{ml: 4, width: 200}}>
```

Comment:
what´s the need of the stack if it only cotains  one single element inside? should it be wrapping the `...map(tag, index)`?

Reply:
also, do not use `index` as `key`

Reply:
@nobeeakon This will be recitified in a future PR with the use of a UUID. Currently, since any updates aren't happening in this current version, it has index as key.